projects/demo/src/modules/experimental/card/examples/15/index.html

```diff
@@ -0,0 +1,30 @@
+<div
+    tuiCard="l"
+    tuiSurface="elevated"
+>
+    <div tuiHeader>
+        <h1 tuiTitle>
+            Title
+            <span tuiSubtitle>Subtitle</span>
+        </h1>
+    </div>
+
+    <progress
+        max="100"
+        new
+        size="m"
+        tuiProgressBar
+        [value]="65"
+    ></progress>
+
+    <div>
```

Comment:
Maybe we should change cards from `div` to `section` and these bottom divs to `footer`? Just a thought, might look better semantically.

Reply:
fixed